LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 [fullname]
```

Comment:
I need an official full name of the company to fill in this place holder.

Reply:
I assume this would be Prodigy Education. @mattbullock you may know better.

Reply:
This should be `Prodigy Education Inc.`

Reply:
Updated!